src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <label>
+      <form
+        class="inputForm top"
+        action="search"
+        method="get"
+        data-qa="big"
+      >
+        <img
+          class="big_img"
+          src="images/Search.svg"
+          alt="img">
+        <input
+        class="input big"
+        type="search"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </form>
+    </label>
+
+    <label>
+      <form
+        class="inputForm"
+        action="search"
+        method="get"
+        data-qa="small"
+      >
+        <img class="small_img"
```

Comment:
add one empty line between same level elements
```suggestion
      >
      
        <img class="small_img"
```